### PR TITLE
HPT-1106: Change metadata property names

### PR DESCRIPTION
**Pull request by rwingiupui**
Change metadata property names:
source -> digital_collection
donor -> owning_institution
funder -> funding
format -> digital_specifications
inventor -> author

### CODE REVIEW

File: app/schemas/plum_schema.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ class PlumSchema < ActiveTriples::Schema
   property :start_canvas, predicate: ::RDF::Vocab::IIIF.hasStartCanvas, multiple: false
   # Newspaper metadata
   property :alternative_title, predicate: ::RDF::Vocab::DC.alternative
+  # property :digital_date, predicate: ::RDF::Vocab::DC.available, multiple: false
   property :digital_date, predicate: ::RDF::Vocab::DC.available, multiple: false
```
Comment (mwoodiupui) [inline app/schemas/plum_schema.rb]: Why was a commented-out duplicate of the next line added?

**Comment by rwingiupui** [comment]
Hi Mark,

 

Sorry, I forgot to remove the extra comments.  I’ll remove them and resubmit the pull request.

 

Rich

**Review by andjsmit — APPROVED** [review]
+1 Work for me. Names changes on edit form, display, and UV More Info as expected.

**Comment by aploshay** [comment]
For me, the larger design question here is, do we:
* change the property names, directly, or
* keep the property names as they are, but have a better configuration layer for where and how things are displayed
This pull request takes the first approach.  An argument for the first approach is that it's nice and simple to have the interface display and the property name be consistent.

Some arguments for the second are that we'd stay more in sync with plum schema and the RDF Vocab gem -- keeping RDF::Vocab::MARCRelators.inv as 'inventor', RDF::Vocab::MARCRelators.aut as 'author' -- and that it may be a feature we want and need for other reasons, anyway.  (This is what I had in mind for HPT-972 and didn't finish before it got moved to the backlog.)

**Comment by andjsmit** [comment]
@aploshay Are you suggesting we hold off on merging this PR for now or that we go with this approach currently but write a new story up (or move HPT-972 up) for an upcoming sprint?

**Pull request merged**